src/applications/common/schemaform/save-in-progress/ApplicationStatus.jsx

```diff
@@ -121,8 +121,8 @@ export class ApplicationStatus extends React.Component {
               cssClass="va-modal-large"
               id="start-over-modal"
               onClose={this.toggleModal}
+              title="Starting over will delete your in-progress form."
```

Comment:
aXe was throwing an error because the modal wasn't correctly labeled, which we handle through a title prop